govinfo.info.yml

```diff
@@ -2,7 +2,5 @@ name: govinfo
 type: module
 description: A Drupal implementation and interface for the govinfo API.
 package: Other
-version: 1.0
-core_version_requirement: ^9
-
-php: 7.4
+version: 1.1
+core_version_requirement: ^9 || ^10
```

Comment:
Drupal 9.0 is no longer available. Make available for Drupal 10 unless we're still working on a D9 site for this.